src/index.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import { BrowserRouter as Router} from 'react-router-dom';
+
 import App from './App';
+
+
```

Comment:
I don't see Any Links to go along with the Routes. We should have Link Tags to allow  us to route to specific components